docs/alps/storage.md

```diff
@@ -18,29 +17,24 @@ These separate storage clusters are on the same Slingshot 11 network as Alps.
 | IOPs         | 1.5M                   | 8.6M read, 24M write   | 200k read, 768k write |
 | file create/s| 374k                   | 214k                   | 97k                 |
 
-
-!!! todo
-    Information about Lustre. Meta data servers, etc.
-
-    * how many meta data servers on Capstor and Iopsstor
-    * how these are distributed between store/scratch
-
-    Also discuss how Capstor and iopstor are used to provide both scratch / store / other file systems
+Capstor and Iopsstor are on the same Slingshot network as Alps, while VAST is on the CSCS Ethernet network.
 
 The mounts, and how they are used for Scratch, Store, and Home file systems that are mounted on clusters are documented in the [file system docs][ref-storage-fs].
 
 [](){#ref-alps-capstor}
 ## Capstor
 
-Capstor is the largest file system, for storing large amounts of input and output data.
+Capstor is the largest file system, and it is meant for storing large amounts of input and output data.
 It is used to provide [scratch][ref-storage-scratch] and [store][ref-storage-store].
 
-!!! todo "add information about meta data services, and their distribution over scratch and store"
+Capstor has 80 Object Storage Servers (OSS), and 6 Metadata Servers (MDS). 
+Two of of these Metadata servers are dedicated for Store, and the remaining four are dedicated for Scratch.
 
 [](){#ref-alps-capstor-scratch}
 ### Scratch
 
 All users on Alps get their own scratch path on Alps, `/capstor/scratch/cscs/$USER`.
+Since Capstor OSSs are made of HDDs, Capstor is a storage well suited for jobs which perform large sequential and parallel read/write operations.
```

Comment:
```suggestion
Since Capstor OSSs are made of HDDs, Capstor is a storage well suited for jobs which perform large sequential and parallel read/write operations.
See the [Scratch documentation][ref-storage-scratch] for more information.
```